docs/developer-docs/production/system-canisters.mdx

```diff
@@ -0,0 +1,24 @@
+# System canisters
+
+## Overview
+
+System canisters are canisters that provide a necessary function to the ICP network. These canisters are controlled by the NNS and are upgraded via proposals. These canisters have static canister IDs that projects can reference. One common example of a system canister that is referenced by projects is the Internet Identity canister. 
+
+## System canisters
+
+| System canister | Canister ID |
+|-----------------|-------------|
+| NNS registry        | rwlgt-iiaaa-aaaaa-aaaaa-cai |
```

Comment:
I suggest linking to any docs on the topics unless we feel that developers will not reference this doc until they have a pretty thorough understanding of ICP